_posts/en/2016-03-11-make-easy-loop-on-array.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+
+title: Make easy loop on array
+tip-number: xx
+tip-username: jamet-julien
+tip-username-profile: https://github.com/jamet-julien
+tip-tldr: Make a elegant loop on array 
+
+categories:
+    - fr
+---
+We often need to build loop, all the conditions can be huge:
+
+```js
+
+var index = 0,
+    aList = ['A','B','C', 'D', 'E']
+
+function change( dir){
```

Comment:
You always have a space after `(`  ... either use it on both sides or remove it.